napari_plugin_manager/qt_plugin_dialog.py

```diff
@@ -711,11 +719,16 @@ def __init__(self, parent=None) -> None:
 
         self._plugin_data = []  # Store plugin data while populating lists
         self.all_plugin_data = []  # Store all plugin data
+        self._filter_texts = []
+        self._filter_idxs_cache = set()
+        self._filter_timer = QTimer(self)
+        self._filter_timer.setInterval(200)  # ms
```

Comment:
Can you explain the rationale for this? It's noticeable as a lag when typing. I tried 100 and it felt better.

Reply:
Hi @psobolewskiPhD,  I think between 100-150 ms is what is a expected time between keypresses, so to avoid triggering a filter for every keystroke when one wants to write a word.

I just bumped it a bit to 200, but 150 should work fine. 100 might be a little on the edge? And yes it is noticeable so maybe we should decrease it

I know it is noticeable, so maybe going for 150 feels better?